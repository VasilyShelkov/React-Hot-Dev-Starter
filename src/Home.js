import React from "react";

export default ({ match }) =>
  <div>
    <h1>Router Home</h1>
    <ul>
      <li>
        <strong>Pathname</strong>: {match.path}
      </li>
    </ul>
  </div>;
