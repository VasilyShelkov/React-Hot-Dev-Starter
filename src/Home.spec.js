import React from "react";
import Home from "./Home";

describe("src/Home.js", () => {
  it("renders the Home page", () => {
    expect(
      renderer.create(<Home match={{ path: "/home" }} />).toJSON()
    ).toMatchSnapshot();
  });
});
