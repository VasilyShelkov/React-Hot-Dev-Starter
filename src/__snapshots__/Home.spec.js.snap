// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/Home.js renders the Home page 1`] = `
<div>
  <h1>
    Router Home
  </h1>
  <ul>
    <li>
      <strong>
        Pathname
      </strong>
      : 
      /home
    </li>
  </ul>
</div>
`;
